styles/prosilver/template/index_body.html

```diff
@@ -65,7 +65,7 @@ <h2>{L_CUSTOMISATION_DATABASE}</h2>
 			</div>
 		</div>
 
-		<div class="dropdown-container sort-option status-sort rightside">
+		<div class="dropdown-container sort-option status-sort rightside"<!-- IF not SHOW_STATUS --> style="display: none;"<!-- ENDIF -->>
```

Comment:
Why wouldn't you just wrap this entire HTML block for the drop down in an `<!-- IF SHOW_STATUS -->` condition? Would be better to just not render it to the template at all than use CSS and JS to hide it.

Reply:
It needs to be done with JS/CSS because if you click on to a different category it reloads the items with AJAX. 

For example, if we wrapped the entire thing in an `IF` block and the first type the user loaded was a type where the user didn't have validate permissions - if that user then clicked on to another type where they did have permissions, the HTML wouldn't exist to toggle the visibility in JS.